extensions/ql-vscode/src/vscode-tests/factories/remote-queries/shared/variant-analysis-submission.ts

```diff
@@ -0,0 +1,20 @@
+import { VariantAnalysisQueryLanguage, VariantAnalysisSubmission } from '../../../../remote-queries/shared/variant-analysis';
+
+export function createMockSubmission(): VariantAnalysisSubmission {
+  return {
+    startTime: 1234,
+    controllerRepoId: 5678,
+    actionRepoRef: 'repo-ref',
+    query: {
+      name: 'query-name',
+      filePath: 'query-file-path',
+      language: VariantAnalysisQueryLanguage.Javascript,
+      pack: 'base64-encoded-string',
+    },
+    databases: {
+      repositories: ['1', '2', '3'],
+      repositoryLists: ['top10', 'top100'],
+      repositoryOwners: ['mona', 'lisa'],
```

Comment:
Would we usually have all three populated? I would expect this to normally only have one of these fields populated.

Reply:
You're right, at the moment we only ever send one and even validate to check that only one of these fields is present in the API. 

However in the future we do intend to make it possible to send combinations of these. 

🤔  I think for now it does makes sense to keep just one of them to be consistent with what the API is doing.

Reply:
I've left `repositories`.